HIP/hip-336.md

```diff
@@ -291,8 +319,68 @@ The allowance adjustment transaction will fail under the following circumstances
 - The token referenced in an NFT allowance is fungible.
 - Duplicate crypto or fungible token allowances for a spender are specified.
 - Duplicate NFT serial number allowances for a spender are specified.
+- A negative NFT serial number is specified.
 - No allowances are specified. 
 
+### Add CryptoDeleteAllowance API
+
+A new Hedera API will be added under the Crypto Service called `CryptoDeleteAllowance`. This function is called by the token owner to delete an existing allowance.
+
+```protobuf
+// new message
+message CryptoWipeAllowance {
+    AccountID owner = 1;
+}
+
+// new message
+message NftWipeAllowance {
+    TokenID token_id = 1;
+    AccountID owner = 2;
+    repeated int64 serial_numbers = 3;
+}
+
+// new message
+message TokenWipeAllowance {
+    TokenID token_id = 1;
```

Comment:
As discussed , In order not to go through the for loop (currently we are doing for fungible token) , we can add spender